pkg/repository/v1_repository_test.go

```diff
@@ -566,3 +570,56 @@ func hash(s string) string {
 
 	return hex.EncodeToString(shaWriter.Sum(nil))
 }
+
+func TestWithMigrate(t *testing.T) {
```

Comment:
Could you add a comment to this test to explain what is happening? It is a bit less obvious than the other tests I think.

Reply:
added by https://github.com/pingcap-incubator/tiup/pull/241/commits/a7f51bf6196078be0b138acb81b9cd3ec08ff8ba